src/main/scala/ch/epfl/bluebrain/nexus/iam/realms/RealmEvent.scala

```diff
@@ -0,0 +1,11 @@
+package ch.epfl.bluebrain.nexus.iam.realms
+
+sealed trait RealmEvent extends Product with Serializable
+
+object RealmEvent {
+
+  final case class RealmCreated(realm: Realm)           extends RealmEvent
```

Comment:
add also `rev: Long` to the `RealmCreated ` event. That's how we have it in KG and ADMIN